iModelJsNodeAddon/presentation/ECPresentationSerializer.cpp

```diff
@@ -1408,37 +1426,29 @@ rapidjson::Document IModelJsBoundQueryValueSerializer::_ToJson(BoundQueryIdSet c
 +---------------+---------------+---------------+---------------+---------------+------*/
 rapidjson::Document IModelJsBoundQueryValueSerializer::_ToJson(BoundECValueSet const& boundECValueSet, rapidjson::Document::AllocatorType* allocator) const
     {
-    auto const& values = static_cast<ECValueVirtualSet const*>(boundECValueSet.GetSet().get())->GetValues();
     rapidjson::Document json(allocator);
     json.SetObject();
     json.AddMember("type", "ValueSet", json.GetAllocator());
-    json.AddMember("valueType", rapidjson::Value(values.empty() ? 0 : GetECValueTypeName((*values.begin()).GetPrimitiveType()).c_str(), json.GetAllocator()), json.GetAllocator());
+
+    Nullable<PrimitiveType> valueType = boundECValueSet.GetValueType();
+    if (valueType.IsNull())
+        json.AddMember("valueType", rapidjson::Value(), json.GetAllocator());
+    else
+        json.AddMember("valueType", rapidjson::Value(PrimitiveTypeAsString(*valueType.get()).c_str(), json.GetAllocator()), json.GetAllocator());

```

Comment:
can use `*valueType` instead of `*valueType.get()`

Reply:
Done.